File: app/src/main/java/vn/loitp/app/activity/customviews/watchwhilelayout/FragmentHost.java
package vn.loitp.app.activity.customviews.watchwhilelayout;

import vn.loitp.app.activity.customviews.watchwhilelayout.utils.WWLMusicDataset;

/**
 * Created by thangn on 2/26/17.
 */

public interface FragmentHost {
    void goToDetail(WWLMusicDataset.DatasetItem item);

    void onVideoCollapse();

    void onVideoFullscreen(boolean selected);
}

File: app/src/main/java/vn/loitp/app/a/customviews/wwlMusic/interfaces/FragmentHost.kt
package vn.loitp.app.a.customviews.wwlMusic.interfaces

import vn.loitp.app.a.customviews.wwlMusic.utils.WWLMusicDataset

interface FragmentHost {
    fun goToDetail(item: WWLMusicDataset.DatasetItem)

    fun onVideoCollapse()

    fun onVideoFullscreen(selected: Boolean)
}
